tests/platform.lua

```diff
@@ -0,0 +1,34 @@
+-- Determine some platform information used for tests
+-- Adapted from: https://github.com/keplerproject/luarocks/blob/master/src/luarocks/cfg.lua
+local platform = {}
```

Comment:
Yeah, true. Again, I "borrowed" the implementation from luarocks.cfg; but they likely use the combination of platforms for various things, whereas the luv tests only really need to know if the os is Windows or not.